File: application/web/src/app/testing/test-data/personage-test-data.ts
import { PersonageDTO } from "../../shared/models/personage-dto";

export class PersonageTestData {
    static okabeRintarou(): PersonageDTO {
        const personageDTO: PersonageDTO = new PersonageDTO();

        personageDTO.firstName = "Okabe";
        personageDTO.lastName = "Rintarou";
        personageDTO.photoUrl = "http://photo.url.com/okabe.rintarou";

        return personageDTO;
    }

    static makiseKurisu(): PersonageDTO {
        const personageDTO: PersonageDTO = new PersonageDTO();

        personageDTO.firstName = "Makise";
        personageDTO.lastName = "Kurisu";
        personageDTO.photoUrl = "http://photo.url.com/makise.kurisu";

        return personageDTO;
    }
}

File: application/core/src/test/java/com/korogi/core/domain/testdata/PersonageTestData.java
package com.korogi.core.domain.testdata;

import static com.korogi.core.domain.Personage.newPersonage;

import com.korogi.core.domain.Personage;

public class PersonageTestData {
    public static Personage.PersonageBuilder okabeRintarou_notPersisted() {
        return newPersonage()
            .firstName("Okabe")
            .lastName("Rintarou")
            .photoUrl("http://photo.url.be");
    }
}